providers/src/airflow/providers/standard/operators/python.py

```diff
@@ -187,7 +188,15 @@ def __init__(
     def execute(self, context: Context) -> Any:
         context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
         self.op_kwargs = self.determine_kwargs(context)
-        self._asset_events = context_get_outlet_events(context)
+
```

Comment:
@Lee-W @uranusjr (cc: @gopidesupavan )  -> can you please take a look?  I think - by looking at the history of changes/matching them to released versions, I figured out how to implement "standard-provider" python to work with past versions of Airflow re assets/datasets - to support various renames that you implemented in the past

2.8, 2.9 -> no datasets/assets support
2.10.* (also 2.11 in the future) -> self._dataset_events = context_get_outlet_events(context)
3.0+ >  self._asset_events = context_get_outlet_events(context)

There was also a rename of context_get_outlet_events (used to be named context_get_dataset_events) - but I believe it's never been relesed.

Could you please review and confirm? 

Was there the _dataset_events support before 2.10.0 ?

Reply:
Just checked. I think you're right 👍